docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# set -e
+
+poetry run gunicorn --bind 0.0.0.0:$PORT todo_app.wsgi:app
```

Comment:
You don't define `$PORT` anywhere. Either give it a default in your Dockerfile with e.g. `ENV PORT=80` or here `--bind 0.0.0.0:${PORT:-80}`